package.json

```diff
@@ -21,12 +21,13 @@
     "preinstall": "npx only-allow pnpm"
   },
   "devDependencies": {
-    "@gadget-client/app-with-no-user-model": "^1.8.0",
-    "@gadget-client/bulk-actions-test": "^1.110.0",
-    "@gadget-client/full-auth": "^1.7.0",
-    "@gadget-client/related-products-example": "^1.862.0",
-    "@gadget-client/zxcv-deeply-nested": "^1.210.0",
-    "@gadget-client/zxcv-simple-relationship": "^1.20.0",
+    "@arethetypeswrong/cli": "^0.15.1",
+    "@gadget-client/app-with-no-user-model": "^1.10.0",
+    "@gadget-client/bulk-actions-test": "^1.113.0",
+    "@gadget-client/full-auth": "^1.9.0",
+    "@gadget-client/related-products-example": "^1.865.0",
+    "@gadget-client/zxcv-deeply-nested": "^1.212.0",
+    "@gadget-client/zxcv-simple-relationship": "^1.23.0",
```

Comment:
Had to get all these clients working first so there were no types mismatches in the tests here 👍